docs/troubleshooting.md

```diff
@@ -17,6 +17,46 @@ Script analysis is provided by the [PSScriptAnalyzer] project on GitHub. If the
 message starts with `[PSScriptAnalyzer]` or if you are getting faulty script diagnostics
 (red and green squiggly lines under PowerShell in scripts) please [open an issue there].
 
+## Double-click isn't selecting the whole variable
+
+Visual Studio Code provides a default set of word separators, that is,
+characters that split words and so affect double-click selections. The editor's
+defaults include both `-` and `$`. In [v2021.5.1] we starting providing a
+default value for PowerShell files that excludes these two symbols. The
+intention of this change was to increase predictability, as double-clicking
+PowerShell symbols would now select the same portion that the extension
+highlights (with the exception of scoped variables due to the continued
+inclusion of `:` as a word separator) as well as align with collected user
+feedback.
```

Comment:
I think the parenthetical here muddies the paragraph. Better to simply explain what's different from the VSCode default.

```suggestion
highlights as well as align with collected user feedback.
```